renderer/src/common/components/settings/registry/registry-type-field.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Select,
+  SelectTrigger,
+  SelectValue,
+  SelectContent,
+  SelectItem,
+} from '../../ui/select'
+import type { UseFormReturn } from 'react-hook-form'
+import {
+  FormField,
+  FormItem,
+  FormLabel,
+  FormDescription,
+  FormControl,
+  FormMessage,
+} from '../../ui/form'
+import type { RegistryFormData } from './schema'
+
+export function RegistryTypeField({
+  isPending,
+  form,
+}: {
+  isPending: boolean
+  form: UseFormReturn<RegistryFormData>
+}) {
+  const { type, source } = form.formState.defaultValues ?? {
+    type: 'default',
+    source: '',
+  }
+
+  return (
+    <FormField
+      control={form.control}
+      name="type"
+      render={({ field }) => (
+        <FormItem className="w-full">
+          <FormLabel required>Registry Type</FormLabel>
+          <FormDescription>
+            Choose between ToolHive default registry, a custom remote registry
+            (HTTP/HTTPS URL), or a custom local registry file.
```

Comment:
This one snuck in. I removed my suggestion because if it's merged then I can't review this PR

Reply:
ops forgot it, thanks